dashboards/jfrog-artifactory-cloud/jfrog-artifactory.json

```diff
@@ -0,0 +1,937 @@
+          }
+        },
+        {
+          "title": "Artifactory calls by HTTP status",
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.stacked-bar"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [
+                  4557083
```

Comment:
Sanitize the dashboard json by following this documentation: https://github.com/newrelic/newrelic-quickstarts/blob/007a527498f612858501400e9d0cc0730a01bed5/CONTRIBUTING.md#dashboards

Reply:
Fixed, thanks!